App.tsx

```diff
@@ -32,169 +33,118 @@ Sentry.init({
-        options={({route}) => ({title: String(route.params.name)})}
-      />
-    </AvalancheCenterTelemetryStack.Navigator>
-  );
-};
+// const AvalancheCenterTelemetryStack = createNativeStackNavigator<HomeStackParamList>();
+// const AvalancheCenterTelemetryStackScreen = ({route}: AvalancheCenterProps) => {
+//   const {center_id} = route.params;
+//   return (
+//     <AvalancheCenterTelemetryStack.Navigator initialRouteName="telemetryStations">
+//       <AvalancheCenterTelemetryStack.Screen
+//         name="telemetryStations"
+//         component={TelemetryScreen}
+//         initialParams={{center_id: center_id}}
+//         options={({route}) => ({title: `${route.params.center_id} Telemetry Stations`})}
+//       />
+//       <AvalancheCenterTelemetryStack.Screen
+//         name="telemetryStation"
+//         component={TelemetryStationScreen}
+//         initialParams={{center_id: center_id}}
+//         options={({route}) => ({title: String(route.params.name)})}
+//       />
+//     </AvalancheCenterTelemetryStack.Navigator>
+//   );
+// };
 
-const TelemetryScreen = ({route}: AvalancheCenterProps) => {
-  const {center_id} = route.params;
-  return (
-    <SafeAreaView style={styles.container}>
-      <TelemetryStationMap center_id={center_id} />
-    </SafeAreaView>
-  );
-};
+// const TelemetryScreen = ({route}: AvalancheCenterProps) => {
+//   const {center_id} = route.params;
+//   return (
+//     <View style={styles.container}>
+//       <TelemetryStationMap center_id={center_id} />
+//     </View>
+//   );
+// };
 
-type TelemetryStationProps = NativeStackScreenProps<StackParamList, 'telemetryStation'>;
-const TelemetryStationScreen = ({route}: TelemetryStationProps) => {
-  const {center_id, source, station_id} = route.params;
-  return (
-    <SafeAreaView style={styles.container}>
-      <TelemetryStationData center_id={center_id} source={source} station_id={station_id} />
-    </SafeAreaView>
-  );
-};
-
-type StackParamList = {
-  avalancheCenterSelection: undefined;
-  avalancheCenterHome: {
-    center_id: string;
-    date: string;
-  };
-  avalancheCenterStack: {
-    center_id: string;
-    date: string;
-  };
-  avalancheCenter: {
-    center_id: string;
-    date: string;
-  };
-  forecast: {
-    center_id: string;
-    forecast_zone_id: number;
-    date: string;
-  };
-  avalancheCenterTelemetryStack: {
-    center_id: string;
-  };
-  telemetryStations: {
-    center_id: string;
-  };
-  telemetryStation: {
-    center_id: string;
-    source: string;
-    station_id: number;
-    name: string;
-  };
-};
-
-// TODO(brian): dig into this. This is somehow suppressing typescript errors in
-// other files - if you comment it out, compilation fails.
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace ReactNavigation {
-    // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    interface RootParamList extends StackParamList {}
-  }
-}
```

Comment:
figured out the magic here - now using explicitly exported `HomeStackParamList` from `routes.ts` and then calling `useNavigation<HomeStackParamList>`